trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/reconciliation-summary-helper.ts

```diff
@@ -36,25 +67,78 @@ const getDistinctReportPeriodsByStartMonth = (reportPeriods: ReportPeriod[]): Re
   return distinctPeriods;
 };
 
+/**
+ * Get the report due date text for the given report due date
+ * @param reportDueDate - The report due date
+ * @returns The due date text
+ */
 export const getDueDateText = (reportDueDate: Date) => {
   const reportIsPastDue = isPast(endOfDay(reportDueDate));
   const formattedReportDueDate = format(reportDueDate, 'd MMMM yyyy');
   return `Reports${reportIsPastDue ? ' were ' : ' '}due to be received by ${formattedReportDueDate}.`;
 };
 
-export const getReportPeriodHeading = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
+/**
+ * Get report period heading with all periods
+ * @param submissionMonth - The submission month
+ * @param reportPeriods - List of report periods
+ * @returns Report period heading with the report periods listed
+ */
+const getReportPeriodHeadingWithAllPeriods = (submissionMonth: IsoMonthStamp, reportPeriods: ReportPeriod[]) => {
   const isCurrentSubmissionMonth = isSameMonth(new Date(submissionMonth), new Date());
 
   const formattedReportPeriods = reportPeriods.map((reportPeriod) => getFormattedReportPeriodWithShortMonth(reportPeriod, true)).join(' and ');
 
   return `${isCurrentSubmissionMonth ? 'Current reporting period' : 'Open reports'}: ${formattedReportPeriods}`;
```

Comment:
@RLavender98 could you also please update this, appreciate this is not in your PR. Thanks

Reply:
Ahh I see, these are subtly different to other prefixes, I will see if I can find a commonisation.

Reply:
I've made an attempt to commonise. It feels a little over-engineered to me but I'm not sure how else to do with constants whilst avoiding repetition of substrings. Let me know what you think!